src/service/viewport-impl.js

```diff
@@ -794,42 +774,28 @@ export class ViewportBindingNaturalIosEmbed_ {
     // found is to reset scrolling on the AMP doc, which overrides natural BODY
     // scrolling with overflow:auto. We need the following styling:
     // html {
-    //   overflow: auto;
     //   -webkit-overflow-scrolling: touch;
     // }
     // body {
-    //   position: absolute;
-    //   overflow: auto;
-    //   -webkit-overflow-scrolling: touch;
+    //   overflow-x: hidden;
     // }
     setStyles(documentElement, {
-      overflowY: 'auto',
       webkitOverflowScrolling: 'touch',
     });
     setStyles(documentBody, {
       overflowX: 'hidden',
-      overflowY: 'auto',
-      webkitOverflowScrolling: 'touch',
-      position: 'absolute',
-      top: 0,
-      left: 0,
-      right: 0,
-      bottom: 0,
     });
 
     // Insert scrollPos element into DOM. See {@link onScrolled_} for why
     // this is needed.
     this.scrollPosEl_ = this.win.document.createElement('div');
     this.scrollPosEl_.id = '-amp-scrollpos';
     setStyles(this.scrollPosEl_, {
-      position: 'absolute',
```

Comment:
Why are these removed? This looks to assume a lot about top-level body layout, such as that padding will always be zero or that no any kind of reordering would be done. And w/o this - does this have a benefit?